recipes/logr/v0.7/conanfile.py

```diff
@@ -0,0 +1,116 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, copy,rm
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc
+import os
+
+required_conan_version = ">=1.50.0"
+
+class LogrConan(ConanFile):
+    name = "logr"
+    description = (
+        "Logger frontend substitution for spdlog, glog, etc "
+        "for server/desktop applications"
+    )
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ngrodzitski/logr"
+    topics = ("logger", "development", "util", "utils")
+    settings = "os", "arch", "compiler", "build_type"
+    package_type = "header-library"
+    options = {
+        "backend": ["spdlog", "glog", "log4cplus", "boostlog", None],
+    }
+    default_options = {
+        "backend": "spdlog",
+    }
```

Comment:
@ngrodzitski Looking the project, it seems like we can have more than one backend at same time. Is it correct? https://github.com/ngrodzitski/logr/blob/main/CMakeLists.txt#L39

Is this more aligned with the project's design? 

```suggestion
    options = {
        "with_spdlog_backend": [True, False],
        "with_glog_backend": [True, False],
        "with_log4cplus_backend": [True, False],
        "with_boostlog_backend": [True, False],
    }
    default_options = {
        "with_spdlog_backend": True,
        "with_glog_backend": True,
        "with_log4cplus_backend": True,
        "with_boostlog_backend": True,
    }
```